realm/realm-library/src/main/cpp/jni_util/log.hpp

```diff
@@ -181,6 +181,10 @@ class CoreLoggerBridge : public realm::util::RootLogger {
     void do_log(Logger::Level, std::string msg) override;
     static CoreLoggerBridge& shared();
 
+    // Only used by the sync client to create it's own copy. In most other cases use
+    // CoreLoggerBridge::shared()
+    CoreLoggerBridge() = default;
```

Comment:
`CoreLoggerBridge::shared()` is commented out and the constructor should be private :(